pkg/bootstrap/docker/openshift/metrics.go

```diff
@@ -186,7 +187,27 @@ func metricsDeployerPod(hostName, imagePrefix, imageVersion string) *kapi.Pod {
 			Env: env,
 		},
 	}
-	return pod
+
+	completions := int32(1)
+
+	deadline := int64(60 * 5)
+
+	meta := kapi.ObjectMeta{
+		Name:      pod.Name,
+		Namespace: infraNamespace,
```

Comment:
you don't need to set a namespace. The client will create it in the right namespace